components/src/components/SimpleContent/Component.ts

```diff
@@ -9,26 +10,43 @@ const ID = 'simplecontent';
 const DISPLAY = 'Static Text';
 
 export default class Component extends (ParentComponent as any) {
-    static schema(...extend) {
-        return ParentComponent.schema({
-            type: ID,
-            label: DISPLAY,
-            key: ID,
-            input: false,
-            html: ''
-        }, ...extend);
-    }
+  static schema(...extend) {
+    return ParentComponent.schema({
+      type: ID,
+      label: DISPLAY,
+      key: ID,
+      input: false,
+      html: ''
+    }, ...extend);
+  }
+
+  public static editForm = editForm;
+
+  static get builderInfo() {
+    return {
+      title: DISPLAY,
+      group: 'simple',
+      icon: 'pencil-square-o',
+      weight: 40,
+      documentation: Constants.DEFAULT_HELP_LINK,
+      schema: Component.schema()
+    };
+  }
 
-    public static editForm = editForm;
-
-    static get builderInfo() {
-        return {
-            title: DISPLAY,
-            group: 'simple',
-            icon: 'pencil-square-o',
-            weight: 40,
-            documentation: Constants.DEFAULT_HELP_LINK,
-            schema: Component.schema()
-        };
+  get defaultSchema() {
+    return Component.schema();
+  }
+
+  addTargetBlankToLinks = (html) => {
+    let result = '';
+    if (html.includes('<a href=')) {
+      result = html.replace(/<a href=/g, '<a target="_blank" href=');
```

Comment:
While this works in the case where the anchor does not already have a target blank in the element, it's possible to have a situation where if a user manually writes in anchor html that has other stuff between the a and href, you'll miss inserting a target blank into that DOM element. Not too sure if there's a way to parse the html DOM string and then search on that instead of doing it via regex, or if that would be less computationally expensive.

Reply:
After entering a anchor tags manually through the component's textfields, it looks like my code and FormIO is able to differentiate between rendered HTML vs the content of the component.